commit 9a64d3cf9fae39337943174fd7d680a62bade2fa
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Date:   Wed Oct 8 11:31:52 2014 -0500

    lxc-test-unpriv: don't clear out /etc/lxc/lxc-usernet
    
    Signed-off-by: Serge Hallyn <serge.hallyn@ubuntu.com>

diff --git a/src/tests/lxc-test-unpriv b/src/tests/lxc-test-unpriv
index 6235901..a27857c 100755
--- a/src/tests/lxc-test-unpriv
+++ b/src/tests/lxc-test-unpriv
@@ -73,7 +73,7 @@ deluser $TUSER && rm -Rf $HDIR || true
 useradd $TUSER
 
 mkdir -p $HDIR
-echo "$TUSER veth lxcbr0 2" > /etc/lxc/lxc-usernet
+echo "$TUSER veth lxcbr0 2" >> /etc/lxc/lxc-usernet
 sed -i '/^lxcunpriv:/d' /etc/subuid /etc/subgid
 
 usermod -v 910000-919999 -w 910000-919999 $TUSER
